packages/esm-patient-chart-app/src/visit/visits-widget/visit.resource.tsx

```diff
@@ -69,6 +80,26 @@ export function usePastVisits(patientUuid: string) {
   };
 }
 
+export function deleteVisit(visitUuid: string) {
```

Comment:
Shouldn't this call the DELETE method?

Reply:
Calling `DELETE` is sufficient to void the visit, though, and much easier.

Reply:
Got it, on it.

Reply:
One last thing, how can I recover a deleted visit?

Reply:
Same thing you're already doing. The `DELETE` endpoint will just void the record.